app/server/companion.ts

```diff
@@ -194,7 +190,7 @@ export function addDbCommand(program: commander.Command,
       if (!process.env.TYPEORM_LOGGING) {
         process.env.TYPEORM_LOGGING = 'true';
       }
-      const connection = reuseConnection || await getOrCreateConnection();
+      const connection = reuseConnection || await createNewConnection();
```

Comment:
Can you add a comment why this was changed? The `reuseConnection` might be a different one from that returned by `getOrCreateConnection`.